mammoth/docx/body_xml.py

```diff
@@ -350,30 +349,50 @@ def break_(element):
             warning = results.warning("Unsupported break type: {0}".format(break_type))
             return _empty_result_with_message(warning)
 
-    
     def inline(element):
+        content_info = {'width': '', 'height': '', 'alt_text': ''}
+        
         properties = element.find_child_or_null("wp:docPr").attributes
+
         if properties.get("descr", "").strip():
-            alt_text = properties.get("descr")
+            content_info['alt_text'] = properties.get("descr")
         else:
-            alt_text = properties.get("title")
+            content_info['alt_text'] = properties.get("title")
+
+        extAttributes = element.find_child("a:graphic") \
+                        .find_child("a:graphicData") \
+                        .find_child("pic:pic") \
+                        .find_child("pic:spPr") \
+                        .find_child("a:xfrm") \
+                        .find_child("a:ext").attributes
```

Comment:
This feels somewhat brittle: in particular, what if one of the children is missing?

Reply:
The `find_child_or_null()` method returns a null object if the child isn't found, which should handle this. Also, are we sure that each of these elements will only occur once, rather than multiple times?

Reply:
I guess there is only one source size? Will have a look at this a bit more, but have used your `find_child_or_null()` function now add added an `if` statement.